scss/main.scss

```diff
@@ -246,6 +246,84 @@ h1,h2,h3 {
   }
 }
 
+.portfolio-hero-background-container {
+  background: $cfa-red;
+}
+
+.portfolio-hero-container {
+  padding-left: 6rem;
+
+  @media (max-width: 768px) {
+    padding-left: 0;
+  }
+
+  @media (min-width: 1440px) {
+    max-width: 1440px;
+    margin: 0 auto;
+  }
+}
+
+.portfolio-hero-contents {
+  max-width: 1440px;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  h1,
+  h4 {
+    color: $white;
+  }
+
+  h4 {
+    @media (max-width: 1058px) {
+      font-size: 0.875rem !important;
+      line-height: 0.875rem;
+      padding-top: 0;
+      margin: 0 0 1rem 0;
+    }
+  }
+
+  h1 {
+    @media (max-width: 1058px) {
+      font-size: 2rem;
```

Comment:
The h1 font size changes to 2rem at 1058px. It shouldn't get smaller until we hit 651px.

<img width="249" alt="Screen Shot 2021-03-31 at 7 12 38 PM" src="https://user-images.githubusercontent.com/9326234/113226168-19f08280-9255-11eb-8482-468ef96887b1.png">